docs/devel/coding-conventions.md

```diff
@@ -33,60 +33,123 @@ Documentation for other releases can be found at
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 Code conventions
   - Bash
+
     - https://google-styleguide.googlecode.com/svn/trunk/shell.xml
-    - Ensure that build, release, test, and cluster-management scripts run on OS X
+
+    - Ensure that build, release, test, and cluster-management scripts run on
+OS X
```

Comment:
puke.  How many different renderings do we expect to work?  Anything more
that GitHub (GFM) ?

On Tue, May 3, 2016 at 5:13 PM, Mike Brown <notifications@github.com> wrote:

> In docs/devel/coding-conventions.md
> <https://github.com/kubernetes/kubernetes/pull/25099#discussion_r61977101>
> :
>
> >      - https://google-styleguide.googlecode.com/svn/trunk/shell.xml
> > -    - Ensure that build, release, test, and cluster-management scripts run on OS X
> > +
> > +    - Ensure that build, release, test, and cluster-management scripts run on
> > +OS X
>
> Depends on the parsing tool. Some tools delete the extra whitespace in
> front of line two and after the new line, but not all tools. The key is the
> line is a section or paragraph if you will, with a particular tab over.
> However if you have spaces but no new section identifier like - or * or 1
> then you are at the mercy of the implementation. This is why on many
> manuals when you look at it at the cols it was new lined at all is good but
> then when you strink it a few lines you get the end of the shorter line
> (tail) as the beginning of the next but then you have an extra new line at
> the end of the tail then the next line with the spaces in front. Having no
> spaces at the front of the next line in the paragraph tells the parsing
> tools that this is a continuing section. Further the number of spaces at
> the beginning does not always correspond to the tabbed section. Getting the
> number of spaces just right is a hit or miss and won't work f or all t
> ools.. Thus better to not not guess.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25099/files/f1b0b315c817c80452752e5861f957583138bae0#r61977101>
>